docs/build_and_load.md

```diff
@@ -467,17 +472,20 @@ This command takes 3 parameters:
 * `--keyfile` - A [tab-delimited](https://en.wikipedia.org/wiki/Tab-separated_values)
   keyfile containing two columns:
 
-| Column | Value                                                                                                                                                                                                                      |
-|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `1`    | Path to FASTA file you would like annotated (this is similar to the text files used to point to the FASTA file paths in the [`agc-compress`](#compress-fasta-files) and [`align-assemblies`](#align-assemblies) commands). |
-| `2`    | Name of the sample that will be (1) appended to each header line and (2) the name of the newly generated FASTA file.                                                                                                       |
-
-  + My example `annotation_keyfile.txt` file would look like this:
-
-    ```
-    data/LineA.fa   LineA
-    data/LineB.fa   LineB
-    ```
+  | Column | Value                                                                                                                                                                                                                      |
+  |--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+  | `1`    | Path to FASTA file you would like annotated (this is similar to the text files used to point to the FASTA file paths in the [`agc-compress`](#compress-fasta-files) and [`align-assemblies`](#align-assemblies) commands). |
+  | `2`    | Name of the sample that will be (1) appended to each header line and (2) the name of the newly generated FASTA file.                                                                                                       |
+
+  + My example `annotation_keyfile.txt` would look like this:
+      ```shell
+      data/Ref.fa Ref
+      data/LineA.fa   LineA
+      data/LineB.fa   LineB
```

Comment:
This example would make more sense if you had a longer initial file name, e.g.
data/Ref-NAM-v5.fa   Ref
data/LineA-1.42.final.fa  LineA

It would then be clearer that the fastasw written are named Ref.fa and LineA.fa

Reply:
I have updated this based on your recommendations.